onebusaway-android/src/main/res/values/donottranslate.xml

```diff
@@ -122,4 +125,28 @@
 
     <!-- General strings -->
     <string name="http_prefix">http://</string>
+
+    <!-- Problem reporting additions -->
+    <string name="ri_append_start">\n\n---------------------\n</string>
+    <string name="ri_append_stop_id">stop_id=%1$s;\n</string>
+    <string name="ri_append_agency_name">agency_name=%1$s;\n</string>
+    <string name="ri_append_trip_id">trip_id=%1$s;\n</string>
+    <string name="ri_append_route_id">route_id=%1$s;\n</string>
+    <string name="ri_append_route_display_name">route_display_name=%1$s;\n</string>
+    <string name="ri_append_trip_name">trip_headsign=%1$s;\n</string>
+    <string name="ri_append_vehicle_id">vehicle_id=%1$s;\n</string>
+    <string name="ri_append_vehicle_location">vehicle_location=%1$s;\n</string>
+    <string name="ri_append_service_date">service_date=%1$s;\n</string>
+    <string name="ri_append_predicted">predicted=%1$s;\n</string>
+    <string name="ri_append_arrival_time">stop_arrival_time=%1$s;\n</string>
+    <string name="ri_append_departure_time">stop_departure_time=%1$s;\n</string>
+    <string name="ri_append_schedule_deviation">schedule_deviation=%1$s min;\n</string>
```

Comment:
I'm wondering if we should add some kind of label here for "early" or "late".

What if we drop the negative sign, and instead have:

* `schedule_deviation=1.550 min early;` // If sign was negative
* `schedule_deviation=1.550 min late;`  // If sign was positive
* `schedule_deviation=0 min;` // If zero (i.e., on time)

This makes it more human readable, and if we're going to parse it we could still convert back from early/late to negative/positive.